util-scripts/start-prod.sh

```diff
@@ -1,3 +1,3 @@
 #stack exec -- cardano-node --report-server http://35.156.164.19:5555 --logs-prefix "logs/abc" --db-path db-abc --peer 35.156.234.240:3000/S-GJ1HDZyHpIS_ZOFBlZsipkGNXy-tmCDtY_CvaiywE= --peer 35.157.148.244:3000/nzcEO6Cv7tKD3REWoWc9vcHMOZEQWFGLX4150bt_cYw= --peer 52.52.249.66:3000/dYGuDj0BrJxCsTC9ntJE7ePT7wUoVdQMH3sKLzQD8bo= --peer 52.8.23.163:3000/8hVd3J5Jc0GCkIV-QeiOkwDqH38QooYGo3iHJnZuIk4= --kademlia-dump-path kademlia-abc.dump $@ --wallet --log-config log-config-prod.yaml
-
-stack exec -- cardano-node --explicit-initial --logs-prefix "logs/abc_prod" --db-path db-abc-prod --peer 13.54.129.104:3000/8hVd3J5Jc0GCkIV-QeiOkwDqH38QooYGo3iHJnZuIk4= --kademlia-dump-path kademlia-abc-prod.dump  --log-config log-config-prod.yaml --wallet
+#stack exec -- cardano-node --explicit-initial --logs-prefix "logs/abc_prod" --db-path db-abc-prod --peer 13.54.129.104:3000/8hVd3J5Jc0GCkIV-QeiOkwDqH38QooYGo3iHJnZuIk4= --kademlia-dump-path kademlia-abc-prod.dump  --log-config log-config-prod.yaml --wallet
+stack exec -- cardano-node --system-start $(date +%s) --report-server http://35.156.164.19:5555 --logs-prefix "logs/abc" --db-path db-abc --peer 35.156.234.240:3000/S-GJ1HDZyHpIS_ZOFBlZsipkGNXy-tmCDtY_CvaiywE= --peer 35.157.148.244:3000/nzcEO6Cv7tKD3REWoWc9vcHMOZEQWFGLX4150bt_cYw= --peer 52.52.249.66:3000/dYGuDj0BrJxCsTC9ntJE7ePT7wUoVdQMH3sKLzQD8bo= --peer 52.8.23.163:3000/8hVd3J5Jc0GCkIV-QeiOkwDqH38QooYGo3iHJnZuIk4= --kademlia-dump-path kademlia-abc.dump $@ --wallet --log-config log-config-prod.yaml
```

Comment:
Production has a fixed start time, so specifying current time here is not correct imo. Take the value from `constants-prod.yaml` instead.